### DISCUSSION TITLE
command line tool: raw files in subfolders

### DISCUSSION THREAD

**Discussion by JM-Bader** [vdemichev/DiaNN#376] [Q&A] [answered]
I would like to process a lot of raw files with the command line tool.
Currently, I have them organized in subfolders by 96well plate. This is convenient as I also want to do execute plate-wise searches. 

However, DIA-NN appears not to find raw files in subfolders (1 level down) of the master raw file folder in the command line tool with the --dir command. Is there a way to enable that? (I could not find a suitable command on the github command line reference page)

**Comment by vdemichev** [comment, marked as answer]
I would suggest to specify individual raw files using the --f command. It's probably not difficult to write a script which generates such --f commands given a set of folders. I will add the option to include subdirectories in --dir in future releases, thank  you for the suggestion!

**Reply by JM-Bader** [reply]
thanks for that idea. 
If it is possible to specify multiple single raw files using --f, it probably also is possible to specify multiple directories using --dir, correct?
(Using --f will make the command very bulky when processing thousands of raw files and cfg files cannot be read in, meaning the raw files will have to be in the main command)

**Reply by vdemichev** [reply]
Yes, indeed, multiple --dir commands should work too, forgot about it :)

**Reply by vdemichev** [reply]
Yes, if for whatever reason --cfg files don't work, then it's easy to exceed max command size.
